translation.rst

```diff
@@ -307,10 +307,10 @@ parts of your application and mocking it in your tests.
 
 Instead of translating a string at the time of creation, you can use a
 "translatable object", which is an instance of the
-:class:`Symfony\\Component\\Translation\\Translatable` class. This object stores
+:class:`Symfony\\Component\\Translation\\TranslatableMessage` class. This object stores
 all the information needed to fully translate its contents when needed::
 
-    use Symfony\Component\Translation\Translatable;
+    use Symfony\Component\Translation\TranslatableMessage;
```

Comment:
And can you maybe split these into a new PR as well? Then we can rebase this PR on 4.4 :)

Reply:
This was fixed in #14371